editor.planx.uk/src/pages/Preview/ReconciliationPage.tsx

```diff
@@ -76,16 +92,24 @@ const ReconciliationPage: React.FC<Props> = ({
             </Box>
           )}
         <Typography variant="h3" component="h2">
-          Review your answers so far
+          Review your {hasSections ? "progress" : "answers"} so far
         </Typography>
-        <SummaryListsBySections
-          breadcrumbs={sortedBreadcrumbs}
-          flow={flow}
-          passport={data?.reconciledSessionData?.passport}
-          changeAnswer={changeAnswer}
-          showChangeButton={false}
-          sectionComponent="h3"
-        />
+        {hasSections ? (
+          <SectionsOverviewList
+            sectionNodes={sectionNodes}
+            sectionStatuses={reconciledSectionStatuses}
+            showChange={false}
```

Comment:
I'm choosing to _not_ link the section title names on the reconciliation page because I think we want to highlight/emphasize the "Continue" button as the singular action (consistent with hiding "change" links on SummaryList on reconciliation, but not review page) - 

On clicking "Continue" from reconciliation, we'll already have any changed nodes queued up to be answered/seen first.